src/Sylius/Bundle/TranslationBundle/DependencyInjection/Compiler/TranslationListenerPass.php

```diff
@@ -24,19 +25,38 @@ class TranslationListenerPass implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container)
     {
-        if (!$container->hasDefinition('sylius.translatable.listener')) {
-            return;
-        }
+        $driver = $container->getParameter('sylius_translation.driver');
```

Comment:
In fact this can be reverted and simplified just one line 40:
```php
$translatableListener->addTag(SyliusResourceBundle::DRIVER_DOCTRINE_MONGODB_ODM === $container->getParameter('sylius_translation.driver') ? 'doctrine_mongodb.odm.event_subscriber' : 'doctrine.event_subscriber', array('priority' => 99));